Sources/ComplexModule/Differentiable.swift

```diff
@@ -27,15 +27,21 @@ where RealType: Differentiable, RealType.TangentVector == RealType {
 
   @derivative(of: real)
   @usableFromInline
-  func _derivativeReal() -> (value: RealType, pullback: (RealType) -> Complex) {
+  func _vjpDerivativeReal() -> (value: RealType, pullback: (RealType) -> Complex) {
```

Comment:
For clarity, I think it's better to omit "derivative" from these function names and just use "jvp" or "vjp":
```suggestion
  func _vjpReal() -> (value: RealType, pullback: (RealType) -> Complex) {
```

Reply:
FYI about a future direction: since `@derivative` functions are never called directly, it would be nice to declare them as "anonymous functions" ([SR-12003](https://bugs.swift.org/browse/SR-12003)), if they're ever added to Swift. That avoids having to pick a name for these functions and prevents them from showing up in code completion.